test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -106,10 +106,9 @@
     ("aten::_scatter_reduce", datetime.date(2022, 1, 31)),
     ("aten::native_multi_head_self_attention", datetime.date(9999, 1, 1)),
     ("aten::_native_multi_head_self_attention", datetime.date(9999, 1, 1)),
-    ("aten::scatter_reduce.two", datetime.date(2022, 3, 15)),
     ("aten::grid_sampler_3d_backward", datetime.date(9999, 1, 1)),
     ("aten::_transform_bias_rescale_qkv", datetime.date(9999, 1, 1)),
-    ("aten::_scatter_reduce.two", datetime.date(9999, 1, 1)),
+    ("aten::scatter_reduce.two", datetime.date(9999, 1, 1)),
```

Comment:
Why this change?

Reply:
Previously this was here when `_scatter_reduce` was moved to  `scatter_reduce`. Now I changed the signature of `scatter_reduce` so I assumed adding it to this dict was necessary